diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -190,6 +190,41 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
         }
     }
 
+    /**
+     * Deletes all spaces and newlines
+     * Used to compare copyrights in yaml and file
+     */
+    private fun String.flatten(): String =
+        replace("\n","")
+        .replace(" ","")
+
+    /**
+     * If it is multiline copyright, this method makes needed indents.
+     * Otherwise, if it is one line copyright, it returns it trimmed.
+     */
+    private fun handleMultilineCopyright(copyrightText: String): String {
+        if (copyrightText.contains("\n")) {
+            return copyrightText
+                    .lines()
+                    .reduce { acc, nextLine ->
+                        when {
+                            !nextLine.any { it in 'a'..'z' || it in 'A'.. 'Z' } -> {
+                                "$acc\n"
+                            }
+                            // checks only first line
+                            acc.isEmpty() -> {
```

Comment:
Isn't the initial value of the accumulator equal to the first element of collection here?

Reply:
Yes, but the first element can be "" (empty). I thought we should not add empty line at the beginning of the copyright text

Reply:
Ah, ok. Then you could call `dropWhile { it.isBlank() }` before `reduce`.

Reply:
Yeah, seems logical

Reply:
Fixed